pkg/chain/ethereum/ethereum.go

```diff
@@ -431,6 +456,20 @@ func (ec *EthereumChain) GetMembers(
 	return keepContract.GetMembers()
 }
 
+// GetHonestThreshold returns keep's honest threshold.
+func (ec *EthereumChain) GetHonestThreshold(
+	keepAddress common.Address,
+) (*big.Int, error) {
+	keepContract, err := ec.getKeepContract(keepAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	return keepContract.HonestThreshold()
```

Comment:
What do you think about returning `uint` from this function? The comparison later can be easier. The maximum group size in the contract is 16 and threshold cannot be higher.

Reply:
I had this thought. I wasn't sure if we want to have the same type as in the contract or want to convert it. I'm fine with returning `uint` here.

Reply:
We do it in the beacon for group size, threshold and all timeouts. Operating on big ints was too painful.

Reply:
This is the only thing I'd change in this PR. Once we eliminate big int from here I am happy to merge.